testapi.pm

```diff
@@ -1539,7 +1539,7 @@ sub eject_cd {
 
 =head2 save_memory_dump
 
-  save_memory_dump(filename => undef, migration_speed => "4096m");
+  save_memory_dump([$filename, $max_downtime]);
```

Comment:
this looks wrong. It's expecting a hash in my reading of the code

Reply:
You are correct :), it's wrong